### PR TITLE
brandonsness Updated unredactor.tsv

### PR DESCRIPTION
**Pull request by brandonsness**
Added my data to the unredactor.tsv file

### CODE REVIEW

File: unredactor.tsv [hunk 1]
```diff
@@ -3517,3 +3517,93 @@ yofhen	testing	Mel Brooks	This movie is one of my all time favorites. I have wat
...
+brandonsness	training	Larry	This is an early one from the boys, but some people may not be satisfied with this one like all the others. I found it to be different somehow than the your average Stooge slapstick. It was more funny for it's jokes rather than the poke in the eye or slap. Watch for a hilarious part when █████ grabs the stethoscope from Moe and sings into it. Moe gives him a good smack. That part made me crack up for a good ten minutes. Another hit for the Stooges.
+brandonsness	training	Moe	This is an early one from the boys, but some people may not be satisfied with this one like all the others. I found it to be different somehow than the your average Stooge slapstick. It was more funny for it's jokes rather than the poke in the eye or slap. Watch for a hilarious part when Larry grabs the stethoscope from ███ and sings into it. ███ gives him a good smack. That part made me crack up for a good ten minutes. Another hit for the Stooges.
```
Comment (cegme) [inline unredactor.tsv:3556]: Multiple redactions. Please fix

File: unredactor.tsv [hunk 2]
```diff
@@ -3517,3 +3517,93 @@ yofhen	testing	Mel Brooks	This movie is one of my all time favorites. I have wat
...
...vitality as she literally punches out the lyrics to the jaunty "Americana." As she socks across the number with appropriate hand gestures, Judy literally seems to be chewing on the words of the song as she screws up her mouth and bugs out her eyes in her intense eagerness to show what she can do.<br /><br />By contrast, Durbin's presentation of "Il Bacio," is far more demure and subdued. Although entirely appropriate for her "classical" selection, Durbin's delivery of Arditi's waltz is much more of the traditional "stand 'n sing" variety than ███████'s physically emotive turn. Nevertheless, though "miniature diva" Deanna does nothing to call attention to herself, with her candid eyes, dazzling smile and artless delivery, she easily holds the screen with "jazz baby" Judy, and their delightful duetting of "Americana" in the short's finale makes one regret all the more that producer Joe Pasternak was never able to realize his dream of pairing Durbin and ███████ in a musical feature film (because Universal refused to loan "Number One Asset" Durbin out).<br /><br />A priceless document of the nascent talents of two remarkable and utterly unique talents. See this one if you get a chance!
```
Comment (cegme) [inline unredactor.tsv:3558]: Multiple redactions

File: unredactor.tsv [hunk 3]
```diff
@@ -3517,3 +3517,93 @@ yofhen	testing	Mel Brooks	This movie is one of my all time favorites. I have wat
...
+brandonsness	training	Trinity	This is one of the anime films from The Animatrix collection, one of nine - the only one done in black and white, and the only one featuring ███████. Richly textured and beautifully rendered in every way, and the animated version of ███████ definitely does her justice. If you're a fan of The Matrix, you will need to put this on your short list.
```
Comment (cegme) [inline unredactor.tsv:3559]: multiple redactions

File: unredactor.tsv [hunk 4]
```diff
@@ -3517,3 +3517,93 @@ yofhen	testing	Mel Brooks	This movie is one of my all time favorites. I have wat
...
...es in this one, maybe even better than his days as "The Virginian." In a way, "Hell To Pay" reminds me of those movies in the 60's where actors you know from so many shows make an appearance. If you're of a certain age, Buck Taylor, Peter Brown and Denny Miller and William Smith provide a "wow" factor because we seldom get to see these icons these days. "Hell To Pay" features screen legends along with newer names in Hollywood. Most notable in the cast of "newbies" is Rachel Kimsey (Rebekah), who I've seen lately on "The Young and The Restless" and Kevin Kazakoff, who plays the angst-ridden Kirby, a war-weary man who's torn between wanting to live and let live or stepping in to "do the right thing." William Gregory Lee is excellent as Chance, Kirby's mischievous and womanizing brother. Katie Keane plays Rachel, Rebekah's sister, a woman who did what was necessary to stay alive but giving up her pride in the process. In a small but memorable role, Jeff Davis plays Mean Joe, a former Confederate with a rather nasty mean streak. I think we'll be seeing more of these fine actors in the future. "Hell To Pay" is a fun movie with a great story to tell grab the popcorn, we're headin' West!.
```
Comment (cegme) [inline unredactor.tsv:3566]: Multiple redactions

File: unredactor.tsv [hunk 5]
```diff
@@ -3517,3 +3517,93 @@ yofhen	testing	Mel Brooks	This movie is one of my all time favorites. I have wat
...
+brandonsness	training	Kevin Kline	IN & OUT, in my opinion, is a very hilarious movie. I thought that Cameron (Matt Dillon) was wrong to say that Howard (███████████) was gay. The part I liked most was Howard's bachelor party. This was because they were cracking jokes about a lot of things, including Barbra Streisand films. I also thought that Emily (Joan Cusack) looked very beautiful in her wedding dress and that Howard looked good in his tuxedo. My favorite quote of IN & OUT is, "Is everybody gay? Is this 'THE TWILIGHT ZONE'?" That was absolutely hilarious! The one character I couldn't stand was Sonya (Shalom Harlow). This was because she was VERY conceited and snobby. In conclusion, I recommend this movie to all you ███████████ fans who have not seen it. Be prepared to laugh HARD and have a good time when you see it.
```
Comment (cegme) [inline unredactor.tsv:3569]: Multiple redactions

File: unredactor.tsv [hunk 6]
```diff
@@ -3517,3 +3517,93 @@ yofhen	testing	Mel Brooks	This movie is one of my all time favorites. I have wat
...
+brandonsness	validation	Adam Sandler	A friend of mine bought this film for £1, and even then it was grossly overpriced. Despite featuring big names such as ████████████, Billy Bob Thornton and the incredibly talented Burt Young, this film was about as funny as taking a chisel and hammering it straight through your earhole. It uses tired, bottom of the barrel comedic techniques - consistently breaking the fourth wall as Sandler talks to the audience, and seemingly pointless montages of 'hot girls'.<br /><br />████████████ plays a waiter on a cruise ship who wants to make it as a successful comedian in order to become successful with women. When the ship's resident comedian - the shamelessly named 'Dickie' due to his unfathomable success with the opposite gender - is presumed lost at sea, Sandler's character Shecker gets his big break. Dickie is not dead, he's rather locked in the bathroom, presumably sea sick.<br /><br />Perhaps from his mouth he just vomited the worst film of all time.
```
Comment (cegme) [inline unredactor.tsv:3573]: Multiple redactions

**Review by cegme — CHANGES_REQUESTED** [review]
There are several places with multiple reductions. Please go over and do quality control and re-request a review

**Comment by brandonsness** [comment]
Those are all the same name/text. Are we only supposed to censor the name once even if it appears in the text twice?

**Comment by cegme** [comment]
> Those are all the same name/text. Are we only supposed to censor the name once even if it appears in the text twice?

Correct. It is okay if the name appears multiple times. There should be only one redaction, still.

**Review by cegme — CHANGES_REQUESTED** [review]
Sorry, I just did a length check. The maximum size for each text needs to be 1024.

**Review by cegme — APPROVED** [review]
:shipit:

**Pull request merged**